iosHyperskillApp/iosHyperskillApp/Sources/Modules/StudyPlan/StudyPlanViewModel.swift

```diff
@@ -3,18 +3,19 @@ import shared
 import SwiftUI
 
 final class StudyPlanViewModel: FeatureViewModel<
-  StudyPlanScreenViewState,
+  StudyPlanScreenFeature.ViewState,
```

Comment:
Interesting, why Kotlin generated a name like this, usually it's a one-long name `StudyPlanScreenFeatureViewState` 😄